src/point_light.cpp

```diff
@@ -25,8 +25,23 @@ bool PointLight::compute_illumination(const glm::vec3 &int_point, const glm::vec
         intensity = intensity_m * (1.0 - (angle / 1.5708));
         return true;
     }
+}
 
+bool PointLight::compute_illumination(const glm::vec3 &int_point, const glm::vec3 &loc_normal,
+                                      const std::vector<std::shared_ptr<Plane>> &plane_list,
+                                      const std::shared_ptr<Plane> &current_plane, glm::vec3 &color, double &intensity) const {
+    glm::vec3 light_dir = glm::normalize(position_m - int_point);
+    double angle = glm::acos(glm::dot(loc_normal, light_dir));
 
+    if (angle > 1.5708) {
```

Comment:
Consider replacing `1.5708` with `glm::half_pi` 
```suggestion
    if (angle > glm::half_pi()) {
```